src/settings-ui/Microsoft.PowerToys.Settings.UI/Views/FancyZonesPage.xaml

```diff
@@ -11,6 +11,7 @@
     AutomationProperties.LandmarkType="Main">
 
     <Page.Resources>
+        <converters:BoolToObjectConverter x:Key="BoolToComboBoxIndexConverter" TrueValue="1" FalseValue="0"/>
```

Comment:
What does a "boolean" have to do in a combobox index? That just doesn't make sense. (if i understand correctly) it totally assumes there can never be more then two options. A combobox can contain many options - that's what it's for,

Reply:
The setting is saved as a boolean, but we need to set the SelectedIndex on the ComboBox. That's why we convert it.